api/lib/infrastructure/repositories/user-repository.js

```diff
@@ -2,6 +2,41 @@ const _ = require('lodash');
 const BookshelfUser = require('../data/user');
 const { AlreadyRegisteredEmailError } = require('../../domain/errors');
 const { NotFoundError } = require('../../domain/errors');
+const User = require('../../domain/models/User');
+const OrganizationAccess = require('../../domain/models/OrganizationAccess');
+const Organization = require('../../domain/models/Organization');
+const OrganizationRole = require('../../domain/models/OrganizationRole');
+
+function _toOrganizationsAccessesDomain(organizationAccessesBookshelf) {
+  return organizationAccessesBookshelf.map((organizationAccessBookshelf) => {
+    return new OrganizationAccess({
+      id: organizationAccessBookshelf.get('id'),
+      organization: new Organization({
+        id: organizationAccessBookshelf.related('organization').get('id'),
+        code: organizationAccessBookshelf.related('organization').get('code'),
+        name: organizationAccessBookshelf.related('organization').get('name'),
+        type: organizationAccessBookshelf.related('organization').get('type'),
+        email: organizationAccessBookshelf.related('organization').get('email')
```

Comment:
est-ce que vous êtes surs qu'il n'y a qu'une seule requête là ? Sinon, extraire dans un fichier

Reply:
Normalement oui :)

"The related method returns a specified relation loaded on the relations hash on the model, or calls the associated relation method and adds it to the relations hash if one exists and has not yet been loaded."